docs/transactions/EIP_transaction_types/eip4844_blob_transaction.md

```diff
@@ -0,0 +1,105 @@
+## EIP-4844 Blob Transaction
```

Comment:
Add one more text paragraph here that should mention what network needs to support in order to be able to run this.

Also did you manage to send blob to Besu or only the transaction without blob?
If no we should also specify that

Reply:
We were able to send blob tx along with blobs, commitments and proof to sepolia network